svg/text/parsing/text-anchor-computed.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="800px">
+  <title>SVG Text: getComputedStyle().textAnchor</title>
+  <metadata>
+    <h:link rel="help" href="https://svgwg.org/svg2-draft/text.html#TextAnchorProperty"/>
+    <h:meta name="assert" content="text-anchor computed value is as specified."/>
+  </metadata>
+  <g id="target"></g>
```

Comment:
Maybe change to…
```suggestion
  <text id="target"></text>
```

It doesn't make any difference, since the resolved/computed value is as specified, but it's logically a bit more structured to test the computed value on an element that the property actually applies to.